### PR TITLE
Add pepsi logs show

### PR DESCRIPTION
**Pull request by julianschuler**
## Introduced Changes

This PR introduces a `pepsi logs show` subcommand for outputting the logs (hulk.out and hulk.err) of one or multiple NAOs.

## ToDo / Known Issues

None.

## Ideas for Next Iterations (Not This PR)

Filtering the log output may be interesting, though this may rather be done with an external tool.

## How to Test

Execute e.g. `./pepsi logs show 21`.

### CODE REVIEW

File: crates/nao/src/lib.rs [hunk 1]
```diff
@@ -169,6 +169,7 @@ impl Nao {
 
         let status = self
             .rsync_with_nao(true)
+            .arg("-q")
```
Comment (okiwi6) [inline crates/nao/src/lib.rs:172]: Prefer --quiet?

File: tools/pepsi/src/ping.rs [hunk 2]
```diff
@@ -24,25 +19,14 @@ pub struct Arguments {
...
-                Duration::from_secs_f32(arguments.timeout),
-            )
-            .await
-            .map(|_| TaskMessage::EmptyMessage);
-            progress.finish_with(ping_state);
-        })
-        .collect::<FuturesUnordered<_>>()
-        .collect::<Vec<_>>()
-        .await;
+    ProgressIndicator::map_tasks(arguments.naos, "Pinging NAO...", |nao_address| async move {
```
Comment (okiwi6) [inline tools/pepsi/src/ping.rs:22]: Nice, didn't know that function

**Review by okiwi6 — CHANGES_REQUESTED** [review]
I don't quite like that error and log are not distinct in the output. Do you think there is some way to improve this?

File: tools/pepsi/src/ping.rs (hunk 2 above)
Comment (julianschuler) [reply tools/pepsi/src/ping.rs:22]: 👍🏾

File: crates/nao/src/lib.rs (hunk 1 above)
Comment (julianschuler) [reply crates/nao/src/lib.rs]: You're right, done.

**Review by okiwi6 — APPROVED** [review]

**Pull request merged**